### PR TITLE
feat: schedule page

### PR DESCRIPTION
**Pull request by chaung844**
Screen Shot 2023-10-15 at 4 01 36 PM
Screen Shot 2023-10-15 at 4 01 48 PM

### CODE REVIEW

File: src/components/schedule/data.js
Comment (menamerai) [inline src/components/schedule/data.js:1]: This is great, though we can probably do this in JSON to solidify it more as data storage

**Review by menamerai — CHANGES_REQUESTED** [review]
Great work! I'd approve this but I've noticed that some other people are also working on importing data and rendering it dynamically like you. For it to me a more standardized process, can you change the data.js over to say, a schedule.json file in the `src/data/` folder and import it from there instead?

**Comment by chaung844** [comment]
> Great work! I'd approve this but I've noticed that some other people are also working on importing data and rendering it dynamically like you. For it to me a more standardized process, can you change the data.js over to say, a schedule.json file in the `src/data/` folder and import it from there instead?

**Review by menamerai — APPROVED** [review]
LGTM

**Pull request merged**